### ISSUE TITLE
how are the training set and test splited?

### ISSUE THREAD

**Issue by WeianMao** [A4Bio/ProDesign#4]

**Comment by gaozhangyang** [comment]
We provided the training set and test set, following the setting of GraphTrans and GVP.  Proteins are partitioned by the CATH topology classification, resulting in 18024 proteins for training, 608 proteins for validation, and 1120 proteins for testing.
